python/py-argcomplete/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-argcomplete
+version             1.9.4
+platforms           darwin
+license             Apache-2
+maintainers         {outlook.com:mohd.akram @mohd-akram} openmaintainer
+
+description         Bash tab completion for argparse
+long_description    ${description}
+
+homepage            https://argcomplete.readthedocs.io
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  13866fe7923744e726c843da97a807616296d2d2 \
+                    sha256  06c8a54ffaa6bfc9006314498742ec8843601206a3b94212f82657673662ecf1 \
+                    size    47471
+
+python.versions     27 37
```

Comment:
personally, I would include 36 here as well, but I will leave that to you to decide...

Reply:
I'm liking the symmetry with 27/37 so I'm going to stick with this for now. Can always change it if another port needs it.

Reply:
Yes, it is easy to add the other versions later, but much harder to remove them.